_topic_maps/_topic_map.yml

```diff
@@ -2226,6 +2218,29 @@ Topics:
   - Name: Serverless overview
     File: about-serverless
 ---
+Name: Machine configuration
+Dir: machine_configuration
+Distros: openshift-enterprise, openshift-origin
+Topics:
+- Name: Machine configuration overview
+  File: index
+- Name: Using machine config objects to configure nodes
+  File: machine-configs-configure
+- Name: Configuring MCO-related custom resources
+  File: machine-configs-custom
+- Name: Understanding node restart behaviors after machine config changes
+  File: machine-config-node-disruption
+- Name: Updated boot images
+  File: mco-update-boot-images
+- Name: Red Hat Enterprise Linux (RHEL) image layering
```

Comment:
Small nit: should this section be titled "Red Hat Enterprise Linux (RHEL) _CoreOS_ image layering"? 

LGTM otherwise!